addons/wowl/static/src/main.ts

```diff
@@ -21,6 +18,13 @@ declare const odoo: Odoo;
     console: window.console,
   });
 
+  // load templates
+  const templatesUrl = `/wowl/templates/${odoo.session_info.qweb}`;
+  const [templates] = await Promise.all([
+    loadFile(templatesUrl),
+    fetchLocalizationParameters(browser, odoo),
```

Comment:
we wait here for localization. Should we?

Reply:
maybe not in the future, but for now it is fine in my opinion

Reply:
i made another subtask to not wait here... but we will see how

Reply:
ok